src/main/java/blue/endless/jankson/api/document/ObjectElement.java

```diff
@@ -167,6 +167,21 @@ public String toString() {
 		w.flush();
 		return w.toString();
 	}
+
+	/**
+	 * Gets the KeyValuePairElement entry if it exists in this object.
+	 * If no element exists with the specified key then null will be returned.
+	 * @param key the key whose corresponding entry should be returned
+	 * @return The entry if it is present, otherwise null.
+	 */
+	public KeyValuePairElement getKeyValuePair(String key) {
+		for (KeyValuePairElement element : entries) {
+			if (Objects.equals(element.key, key)) {
+				return element;
+			}
+		}
+		return null;
```

Comment:
This is inconsistent with other parts of the API, I'd like to avoid returning null, ever, period. There are two possible remedies:
- Return a null-projection, i.e., a synthetic KeyValuePairElement that maps to PrimitiveElement.ofNull() (less desirable - would be somewhat surprising)
- Change the method to return an Optional&lt;KeyValuePairElement&gt; (preferred)

With the implementation of the Map interface, obviously we can't avoid returning null as it is specified in the interface's behavior. But here we can do better.

Reply:
Oki, I'll use `Optional` route instead.

Reply:
Done!